libgalois/test/graph-predicates.cpp

```diff
@@ -24,7 +25,8 @@ TestIsApproximateDegreeDistributionPowerLaw() {
         !katana::analytics::IsApproximateDegreeDistributionPowerLaw(*g.get()));
   }
   {
-    auto g = katana::PropertyGraph::Make(rmat10InputFile);
+    struct tsuba::RDGLoadOptions opts;
+    auto g = katana::PropertyGraph::Make(rmat10InputFile, opts);
```

Comment:
FWIW you can just do `Make(file, tsuba::RDLoadOptions())`

Reply:
Why?

Reply:
nit: it's more compact.

Reply:
I meant why is it allowed? I assumed that a static class function always had to be called as `Class::Func()`.

Reply:
I'm confused. I meant this:
```cpp
    auto g = katana::PropertyGraph::Make(rmat10InputFile, tsuba::RDGLoadOptions());
```
instead of this:
```cpp
    tsuba::RDGLoadOptions opts;
    auto g = katana::PropertyGraph::Make(rmat10InputFile, opts);
```

Reply:
OOOooohhhh. I thought you were saying to drop the `katana::PropertyGraph::`. That is more compact, I'm happy to change it.

Reply:
I think you can actually write `katana::PropertyGraph::Make(rmat10InputFile, {})` which I prefer even more. The type is implied from the context anyway.